test/test.js

```diff
@@ -797,24 +797,71 @@ describe("Pv should work", () => {
     function testPv(pvCode, fileName, chatBody) {
         stub_console();
         sinon.useFakeTimers();
+
         let fake_socket = new FakeSocket();
         let fake_api = new FakeAPI();
         fake_api.request({ path: '/foo' }, () => { });
         sinon.stub(https, 'request').callsFake(fake_api.request);
+
         let fake_gtp = new FakeGTP();
         sinon.stub(child_process, 'spawn').returns(fake_gtp);
+
         let conn = new connection.Connection(() => { return fake_socket; }, config);
+
         const game = sinon.spy();
         game.sendChat = sinon.spy();
         game.processing = true;
         game.state = { width: 19, moves: { length: 2 } };
+
         config.ogspv = pvCode;
+
         new Bot(conn, game, config.bot_command);
```

Comment:
independently from the persist PR, is something missing in this line?
ex: https://github.com/online-go/gtp2ogs/blob/devel/test/test.js#L757